packages/@o3r/design/src/core/design-token/parsers/design-token.parser.ts

```diff
@@ -41,6 +42,9 @@ const getCssRawValue = (variableSet: DesignTokenVariableSet, {node, getType}: De
   } as typeof node;
 
   switch (checkNode.$type) {
+    case 'string': {
```

Comment:
You could add some unit-test to cover these and for us to understand what kind of sanitization we expect